pkgs/build-support/kernel/modules-closure.sh

```diff
@@ -22,10 +22,13 @@ echo "kernel version is $version"
 closure=
 for module in $rootModules; do
     echo "root module: $module"
-    deps=$(modprobe --config no-config -d $kernel --set-version "$version" --show-depends "$module" \
-        | sed 's/^insmod //') \
-        || if test -z "$allowMissing"; then exit 1; fi
+    # Obtain the module dependencies with modprobe.
+    rawdeps=$(modprobe --config no-config -d "$kernel" --set-version "$version" --show-depends "$module") \
+     || if test -z "$allowMissing"; then exit 1; fi
+
     if [[ "$deps" != builtin* ]]; then
```

Comment:
`$deps` seems to be used before it being declared now

Reply:
Indeed, should be `rawdeps` now.